src/gui/mzroll/isotopeswidget.cpp

```diff
@@ -327,7 +328,7 @@ QString IsotopeWidget::groupIsotopeMatrixExport(PeakGroup* group, bool includeSa
 			MatrixXf MM = _mw->getIsotopicMatrixIsoWidget(group);
 			for (int i=0; i < isotopes.size(); i++ ) {
 				QStringList groupInfo;
-                groupInfo << tag + " " + QString(isotopes[i]->tagString.c_str());
+                groupInfo << tag + " | " + QString(isotopes[i]->tagString.c_str());
```

Comment:
If you find peaks without isotopes and then copy details from peaks table , the compound name is copied with "|". eg. 
" Pyridoxamine | 	21918.21	22437.19	16426.92	24983.37"
I think that happens because  isotopes size is never zero.
If you think it should be fixed now, then do it else _ship it!_

Reply:
@rish9511 fixed it